crates/autopilot/src/solvable_orders.rs

```diff
@@ -248,6 +249,7 @@ impl SolvableOrdersCache {
             balances: new_balances,
         };
 
+        tracing::debug!(%block, "updated current cached auction");
```

Comment:
For improved grep-ability I would also add the auction `id` here.
Then you'll see all the useful information by simply searching for the auction id you are investigating.

Reply:
Unfortunately, we don't always have the `id` available here. I can log an `Option<ID>` though.